src/qibolab/qubits.py

```diff
@@ -71,17 +71,11 @@ class Qubit:
     twpa: Optional[Channel] = None
     drive: Optional[Channel] = None
     flux: Optional[Channel] = None
-    flux_coupler: Optional[List["Qubit"]] = None
+    flux_coupler: Optional[Dict[QubitId, "Qubit"]] = field(default_factory=dict)
```

Comment:
I thought it makes more sense to have the `flux_coupler` in the `QubitPair` instead of `Qubit`, since it is used to control the interaction between two qubits (a pair). For the current IQM chip it doesn't make a difference because qubits are one-to-one with pairs, but for other chips with more connections it may make a difference.

Also for a readability we could leave an empty line from the channel, since `flux_coupler` is not a `Channel`:
```suggestion

    flux_coupler: Optional[Dict[QubitId, "Qubit"]] = field(default_factory=dict)
```
If all couplers are flux, we could even rename it to just `coupler` for simplicity.

Reply:
It makes more sense in the `QubitPair` but I was trying to get things to run with the current chevron that doesn't use `QubitPair` so I can't access a coupler from its qubits if I put it there